lib/wallet/wallet.js

```diff
@@ -1559,6 +1559,87 @@ class Wallet extends EventEmitter {
     }
   }
 
+  /**
+   * Make a batch open MTX.
+   * @param {Array} names
+   * @param {Number|String} acct
+   * @param {Boolean} force
+   * @returns {MTX}
+   */
+
+  async makeBatchOpen(names, force, acct) {
```

Comment:
Can you rewrite all the other `makeOpen`/`createOpen` etc calls so that they are a special case of `xxxBatchOpen`? I want to avoid having bug fixes in one not translate to bug fixes in the other. Perhaps something like:

```javascript
async makeOpen(name, force, acct) {
  const { mtx, errors, isAllError } = await makeBatchOpen([name], force, acct);
  if (errors.length > 0) {
    throw new Error(errors[0].error);
  }

  return mtx;
}
```